src/global.ts

```diff
@@ -158,6 +158,26 @@ export interface Auth0ClientOptions extends BaseLoginOptions {
    */
   useRefreshTokens?: boolean;
 
+  /**
+   * If true, fallback to the legacy technique of using a hidden iframe and the `authorization_code` grant with `prompt=none` when unable to use refresh tokens.
```

Comment:
I wouldn't describe `web_message` as 'legacy'

Reply:
I stole it from the comment above from useRefreshTokens 😬

Reply:
Haha, I bet that was me...